Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -815,6 +815,26 @@ def get_user_info(self) -> Dict[str, Any]:
             url_suffix=f'rest/api/{self.api_version}/myself'
         )
 
+    def get_user(self, identifier: str) -> Dict[str, Any]:
+        """Gets the user from Jira via API
+
+        :param identifier: The URL parameter used to identify the user,
+                           i.e. `f'key={key}'` or `f'username={username}'`
+        :type identifier: str
+        :return: The user's information as returned by the API
+        :rtype: Dict[str, Any]
+        """
+        j_res = self.http_request(
+            'GET',
+            url_suffix=f'rest/api/2/user?{identifier}',
```

Comment:
Lets stay consistent with the endpoints used in the rest of the commands, for reference see: 
https://github.com/demisto/content/blob/abbd5d40ae4e4837ba454bea853b1c01ab67c388/Packs/Jira/Integrations/JiraV3/JiraV3.py#L243

https://github.com/demisto/content/blob/abbd5d40ae4e4837ba454bea853b1c01ab67c388/Packs/Jira/Integrations/JiraV3/JiraV3.py#L525

Reply:
Fixed